src/app/pages/HomePage/index.test.tsx

```diff
@@ -213,6 +213,37 @@ describe('Home Page', () => {
 
       expect(getBootstrapScript()).toBeTruthy();
     });
+    it('should display the MPU ad in the correct location', () => {
+      const { container } = render(
+        <BrowserRouter>
+          {/* @ts-expect-error suppress pageData prop type conflicts due to missing imageAlt on selected historical test data for curations */}
+          <HomePage pageData={homePageData} />
+        </BrowserRouter>,
+        {
+          service: 'kyrgyz',
+          toggles: {
+            ads: { enabled: true },
+          },
+          showAdsBasedOnLocation: true,
+        },
+      );
+      const sections = container.querySelectorAll(`section`);
+      const sectionIds: (string | null)[] = [];
+      sections.forEach((section: HTMLElement) => {
+        sectionIds.push(
+          section.getAttribute('aria-labelledby') ||
+            section.getAttribute('data-e2e'),
+        );
```

Comment:
I think we might be able to do this a little bit better.

**Note to reviewers:** we originally used the `forEach` because the `NodeList` did not support the `map` function.

```suggestion
      const sectionIds: (string | null)[] = Array.from(sections).map((section) => 
      section.getAttribute('aria-labelledby') ||
            section.getAttribute('data-e2e'));
```